web/src/Web.App/Controllers/SchoolController.cs

```diff
@@ -18,9 +17,14 @@ public class SchoolController(
     : Controller
 {
     [HttpGet]
-    public async Task<IActionResult> Index(string urn)
+    public async Task<IActionResult> Index(
+        string urn,
+        [FromQuery(Name = "comparator-generated")] bool? comparatorGenerated)
```

Comment:
Possible future improvement - use TempData rather than query param.

Reply:
I think I'll leave this one as-is for the moment, as it will require change to the polling which is handled client side.